OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -179,14 +179,14 @@ public int GetMovementClass(TileSet tileset)
 
 		static bool IsMovingInMyDirection(Actor self, Actor other)
 		{
-			if (!other.IsMoving()) return false;
-
 			var selfMobile = self.TraitOrDefault<Mobile>();
 			if (selfMobile == null) return false;
 
 			var otherMobile = other.TraitOrDefault<Mobile>();
 			if (otherMobile == null) return false;
 
+			if (!otherMobile.IsMoving) return false;
```

Comment:
While we're here, could we get the codestyle right?
```
if (!otherMobile.IsMoving)
	return false;
```
or even (although not codestyle)
```
var otherMobile = other.TraitOrDefault<Mobile>();
if (otherMobile == null || !otherMobile.IsMoving)
	return false;
```